docs/operational_guide/placement_changes.md

```diff
@@ -0,0 +1,133 @@
+
+#### Endpoint
+
+This value should be in the form of <M3DB_HOST_NAME>:<M3DB_NODE_LISTEN_PORT> and identifies how network requests should be routed to this particular node in the placement.
+
+#### Hostname
+
+This value should be in the form of <M3DB_HOST_NAME> and identifies the address / host name of the M3DB node.
+
+#### Port
+
+This value should be in the form of <M3DB_NODE_LISTEN_PORT> and identifies the port over which this M3DB node expects to receive traffic (defaults to 9000).
+
+### Placement Operations
+
+The instructions below all contain sample curl commands, but you can always review the API documentation by navigating to
+
+`http://<M3_COORDINATOR_HOST_NAME>:<CONFIGURED_PORT(default 7201)>/api/v1/openapi` or our [online API documentation](https://m3db.io/openapi/).
+
+#### Placement Initialization
+
+Send a POST request to the `/api/v1/placement/init` endpoint
+
+```bash
+curl -X POST localhost:7201/api/v1/placement/init -d '{
+    "num_shards": <DESIRED_NUMBER_OF_SHARDS>,
+    "replication_factor": <DESIRED_REPLICATION_FACTOR>(recommended 3),
+    "instances": [
+        {
+            "id": "<NODE_1_ID>",
+            "isolation_group": "<NODE_1_ISOLATION_GROUP>",
+            "zone": "<ETCD_ZONE>",
+            "weight": <NODE_WEIGHT>,
+            "endpoint": "<NODE_1_HOST_NAME>:<NODE_1_PORT>",
+            "hostname": "<NODE_1_HOST_NAME>",
+            "port": <NODE_1_PORT>
+        },
+        {
+            "id": "<NODE_2_ID>",
+            "isolation_group": "<NODE_2_ISOLATION_GROUP>",
+            "zone": "<ETCD_ZONE>",
+            "weight": <NODE_WEIGHT>,
+            "endpoint": "<NODE_2_HOST_NAME>:<NODE_2_PORT>",
+            "hostname": "<NODE_2_HOST_NAME>",
+            "port": <NODE_2_PORT>
+        },
+        {
+            "id": "<NODE_3_ID>",
+            "isolation_group": "<NODE_3_ISOLATION_GROUP>",
+            "zone": "<ETCD_ZONE>",
+            "weight": <NODE_WEIGHT>,
+            "endpoint": "<NODE_3_HOST_NAME>:<NODE_3_PORT>",
+            "hostname": "<NODE_3_HOST_NAME>",
+            "port": <NODE_3_PORT>
+        },
+    ]
+}'
+```
+
+#### Adding a Node
+
+Send a POST request to the `/api/v1/placement` endpoint
+
+```bash
+curl -X POST <M3_COORDINATOR_HOST_NAME>:<M3_COORDINATOR_PORT(default 7201)>/api/v1/placement -d '{
+  "instances": [
+    {
+      "id": "<NEW_NODE_ID>",
+      "isolationGroup": "<NEW_NODE_ISOLATION_GROUP>",
+      "zone": "<ETCD_ZONE>",
+      "weight": <NODE_WEIGHT>,
+      "endpoint": "<NEW_NODE_HOST_NAME>:<NEW_NODE_PORT>(default 9000)",
+      "hostname": "<NEW_NODE_HOST_NAME>",
+      "port": <NEW_NODE_PORT>
+    }
+  ]
+}'
+```
+
+#### Removing a Node
+
+Send a DELETE request to the `/api/v1/placement/<NODE_ID>` endpoint.
```

Comment:
After sending a DELETE, can the node be removed right away or do we need to wait for the cluster to rebalance?